.github/workflows/ci.yaml

```diff
@@ -19,6 +19,41 @@ env:
   PYTHON_VERSION: "3.11" # to build abi3 wheels
 
 jobs:
+  determine_changes:
+    name: "Determine changes"
+    runs-on: ubuntu-latest
+    outputs:
+      linter: ${{ steps.linter.outputs.any_changed }}
+      formatter: ${{ steps.formatter.outputs.any_changed }}
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: tj-actions/changed-files@v37
```

Comment:
Hi @MichaReiser, 🙌 author of [tj-actions/changed-files](https://github.com/tj-actions/changed-files) here. I just came across your workflow and couldn't help but notice a neat little tweak you could make. We've recently introduced a new `files_yaml` input which supports multiple filters in a single step, while still achieving the same results.

I think you might find it pretty handy! For more details, feel free to have a look at our [documentation](https://github.com/tj-actions/changed-files#using-local-git-history). 

Should you have any questions or need further assistance, please don't hesitate to ask. Have a fantastic rest of your day!

Reply:
Oh wow, this is first-class user support <3

This is cool. Let me create a quick PR for it